### DISCUSSION TITLE
Cant import "modality" from intensity_normalization.typing

### DISCUSSION THREAD

**Discussion by Shakka1511** [jcreinhold/intensity-normalization#64] [Q&A] [answered]
Everytime I try to import the "modality" using 

`from intensity_normalization.typing import modalities`

it gives an import error 

> ImportError: cannot import name 'modality' from 'intensity_normalization.typing' (C:\Users\T1463\.conda\envs\py39\lib\site-packages\intensity_normalization\typing.py

Using pythton 3.9

Is there any way to solve this?

**Comment by jcreinhold** [comment, marked as answer]
You should be able to import it by typing: `from intensity_normalization.typing import Modality`. Note the capital `M`.

**Reply by Shakka1511** [reply]
Thank you for the fast response. Cannot believe I missed this... Quick question does the nyul normalizer also work for nrrd's?

**Reply by jcreinhold** [reply]
I didn't test it out, but it might. Try and see. If it fails, convert the image to NIfTI or something else.

**Reply by Shakka1511** [reply]
Does work fine so far once converted into array and removing the additional dimension by converting the NRRD into separate slides.
